conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -1,12 +1,16 @@
 # Copyright (c) 2018-2024, NVIDIA CORPORATION.
 
 {% set data = load_file_data("pyproject.toml") %}
+{% set version = load_file_regex(
+    load_file="pynvjitlink/VERSION",
+    regex_pattern="(?P<value>.*)"
+)[0] %}
 {% set project_data = data.get("project") %}
 {% set project_urls = project_data["urls"] %}
 
 package:
   name: pynvjitlink
-  version: {{ project_data["version"] }}
+  version: {{ version }}
```

Comment:
The version field is left blank now as it is defined dynamically. Fortunately we loaded it from the file already, so we can just use it here